tests/integration/helpers.bash

```diff
@@ -620,3 +621,52 @@ function requires_kernel() {
 		skip "requires kernel $1"
 	fi
 }
+
+function requires_idmap_fs() {
+	local fs
+	fs=$1
+
+	# We need to true it to avoid CI failure as this binary may return with
+	# something different than 0.
+	stderr=$($FS_IDMAP "$fs" 2>&1 >/dev/null || true)
+
+	case $stderr in
+	*invalid\ argument)
+		skip "$fs underlying file system does not support ID map mounts"
+		;;
+	*operation\ not\ permitted)
+		if uname -r | grep -q el9; then
+			# centos kernel 5.14.0-200 contains the following snippet in
+			# can_idmap_mount():
```

Comment:
Christian shared a link, maybe we can put the link here instead of a c&p of the code?

From [this comment](https://github.com/opencontainers/runc/pull/3717#discussion_r1113374333), the code is here:

https://gitlab.com/redhat/centos-stream/src/kernel/centos-stream-9/-/merge_requests/131
https://gitlab.com/redhat/centos-stream/src/kernel/centos-stream-9/-/commit/c93c728360fd10a0dc65